internal/authtoken/authtoken.go

```diff
@@ -115,22 +121,22 @@ func EncryptToken(ctx context.Context, kmsCache *kms.Kms, scopeId, publicId, tok
 
 	marshaledS1Info, err := proto.Marshal(s1Info)
 	if err != nil {
-		return "", fmt.Errorf("error marshaling token info: %w", err)
+		return "", errors.Wrap(err, op, errors.WithCode(errors.Encode))
 	}
 
 	tokenWrapper, err := kmsCache.GetWrapper(ctx, scopeId, kms.KeyPurposeTokens)
 	if err != nil {
-		return "", fmt.Errorf("unable to get wrapper: %w", err)
+		return "", errors.Wrap(err, op, errors.WithMsg("unable to get wrapper"))
 	}
 
 	blobInfo, err := tokenWrapper.Encrypt(ctx, []byte(marshaledS1Info), []byte(publicId))
 	if err != nil {
-		return "", fmt.Errorf("error encrypting token: %w", err)
+		return "", errors.Wrap(err, op, errors.WithCode(errors.Encrypt))
 	}
 
 	marshaledBlob, err := proto.Marshal(blobInfo)
 	if err != nil {
-		return "", fmt.Errorf("error marshaling encrypted token: %w", err)
+		return "", errors.Wrap(err, op, errors.WithCode(errors.Encode))
```

Comment:
Probably best to add  `errors.WithMsg("marshaling encrypted token")` and `errors.WithMsg("marshaling token info")` to here and line 124 to remove the ambiguity between which error occured